### ISSUE TITLE
Increase size of search button icon

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#1334]
The search button icon is abnormally tiny:

<img width="404" alt="Screen Shot 2022-11-17 at 5 02 28 PM" src="https://user-images.githubusercontent.com/101482/202585831-8562c294-3030-46f9-8793-7944bf4884d2.png">

I believe this is because we are doing this in ArcLight, which overrides the icon size in Blacklight:

```
.blacklight-icons svg {
  height: 1rem;
  width: 1rem;
}
```

I'm guessing we did that to simplify getting all of the box, folder, file, etc icons used in ArcLight to a more appropriate, smaller size for the contexts in which we use them. The problem is that that rule is overly-broad and catches the search box icon as well.

Basically, I'd suggest we want the search button icon to be the same size as it is in Blacklight and not be independently sized in ArcLight. (The default display in Blacklight is to also show the "Search" label on the button, but I'm personally fine with the way we don't show that in ArcLight.)
